src/ploomber/spec/dagspec.py

```diff
@@ -322,6 +321,12 @@ def _init(self, data, env, lazy_import, reload, parent_path,
 
         # there is no env.yaml
         else:
+            # wrong extension of env.yml or env.{name}.yml found
+            if default.try_to_find_env_yml(path_to_spec=self._path):
+                raise DAGSpecInitializationError(
+                    'Error: found env file ends with .yml, '
+                    'which is a format that we do not '
+                    'support currently. Change it to .yaml instead.')
```

Comment:
We need to give here a link to the docs (for more context) probably here: https://docs.ploomber.io/en/latest/get-started/basic-concepts.html

Reply:
I don't think there's a need to show the docs, because the fix is simple, rename env.yml to env.yaml. I'd only suggest changing the error message because it's not a different format, it's just a different extension. 

```
'Error: found env file ends with .yml. Change the extension to .yaml.'
```

Reply:
Yup we should isolate the issues, agree.